### PR TITLE
Add contour layer example in website folder

### PR DESCRIPTION
**Pull request by manishmz**
For #3284
- Added contour layer example in the website folder
- Added options to display iso-lines and iso-bands
- Data set is taken from 3d-heatmap source (https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv)

### CODE REVIEW

**Comment by manishmz** [comment]
Contour Layer demo video [link](https://www.loom.com/share/a3c38a2858b34f998ccb767ad451f50b)

File: yarn.lock [hunk 1]
```diff
@@ -11176,4 +11176,4 @@ yauzl@2.4.1:
   resolved "https://registry.yarnpkg.com/yauzl/-/yauzl-2.4.1.tgz#9528f442dab1b2284e58b4379bb194e22e0c4005"
   integrity sha1-lSj0QtqxsihOWLQ3m7GU4i4MQAU=
   dependencies:
-    fd-slicer "~1.0.1"
+    fd-slicer "~1.0.1"
```
Comment (xintongxia) [inline yarn.lock]: Why remove the last line?

Comment (manishmz) [reply yarn.lock]: Not removed intentionally, after installing packages yarn.lock got changed and added in check in.

Comment (xintongxia) [reply yarn.lock]: Could you revert the change since you do not actually change anything?

Comment (manishmz) [reply yarn.lock]: Done

File: examples/website/contour/app.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  zoom: 6.6,
+  minZoom: 5,
+  maxZoom: 15,
+  pitch: 40.5,
+  bearing: -27.396674584323023
+};
+
+const THRESHOLD = {
+  MIN: 1,
+  MAX: 20
```
Comment (1chandu) [inline examples/website/contour/app.js]: MAX value is very less, depending on cell size, weight of cell could be more than 20. Ideally it should be max 32 bit float, but I understand that makes slider too sensitive. Lets make it 500 and see.

File: examples/website/contour/app.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+export class App extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      thresholdValue1: 5,
+      thresholdValue2: 10,
+      cellSize: 1000
+    };
+    this.handleCellSizeChange = this.handleCellSizeChange.bind(this);
```
Comment (1chandu) [inline examples/website/contour/app.js]: Lets use same convention for both function and use `onCellSizeChange` and `onThresholdChange`.

File: examples/website/contour/app.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    const {thresholdValue1, thresholdValue2, cellSize} = this.state;
+    return [
+      new ContourLayer({
+        id: 'contourlayer',
+        coverage,
+        data,
+        getPosition: d => d,
+        opacity: 1,
+        contours: [
+          {threshold: [THRESHOLD.MIN, thresholdValue1], color: [1, 152, 189]},
```
Comment (1chandu) [inline examples/website/contour/app.js]: Lets use these color, in this order `  [255, 255, 178], [253, 141, 60], [189, 0, 38]`

File: examples/website/contour/app.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    this.setState({cellSize});
+  }
+
+  _renderLayers() {
+    const {data, coverage = 1} = this.props;
+    const {thresholdValue1, thresholdValue2, cellSize} = this.state;
+    return [
+      new ContourLayer({
+        id: 'contourlayer',
+        coverage,
```
Comment (1chandu) [inline examples/website/contour/app.js]: there no `coverage` prop, can be removed.

File: examples/website/contour/app.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        data,
+        getPosition: d => d,
+        opacity: 1,
```
Comment (1chandu) [inline examples/website/contour/app.js]: Unless default `opacity: 08` looks bad, lets remove it.

File: examples/website/contour/app.js (hunk 2 above)
Comment (manishmz) [reply examples/website/contour/app.js]: I tried 500 value for max threshold but it is too high that other isoband are not visible. I am updating to 50.

Comment (1chandu) [reply examples/website/contour/app.js]: All the thresholds you are setting are non overlapping, changing it to 500, should only affect one of the contours.

**Comment by 1chandu** [comment]
I also think the demo should be moved to `tests/apps` folder, where it can be polished and then moved back to website folder once ready.

File: examples/website/contour/app.js (hunk 5 above)
Comment (manishmz) [reply examples/website/contour/app.js]: Done

File: examples/website/contour/app.js (hunk 4 above)
Comment (manishmz) [reply examples/website/contour/app.js]: Done

File: examples/website/contour/app.js (hunk 3 above)
Comment (manishmz) [reply examples/website/contour/app.js]: Done

File: examples/website/contour/app.js (hunk 2 above)
Comment (manishmz) [reply examples/website/contour/app.js]: Done

**Comment by manishmz** [comment]
demo link: https://www.loom.com/share/9628d6db1fed46aa8b7c929fdc159817

**Pull request merged**